package.json

```diff
@@ -8,6 +8,7 @@
     "prebuild": "npm run lint && rm -rf lib/*",
     "build": "tsc",
     "lint": "tslint -c tslint.json -e 'node_modules/**/*' '**/*.ts'",
+    "format": "node ./node_modules/prettier/bin-prettier.js src/* --write && [ ! \"$(git status src/ --porcelain)\" ]",
```

Comment:
Usually you can replace `node ./node_modules/prettier/bin-prettier.js` with just `prettier`, if prettier is installed as a dependency or devDependency.

Additionally `src/*` should be replaced with `'src/*'`, which ensures you get the prettier javascript glob implementation instead of the shell globbing, which varies a lot between different OSs.

Reply:
Done:)